.github/settings.yml

```diff
@@ -25,4 +25,4 @@ repository:
   description: "Terraform module to create and work with IBM watsonx.governance."
```

Comment:
Let's go ahead with `watsonx Governance` everywhere instead of `watsonx.governance` just to ensure we are using one name.
Can you please make this change wherever applicable in code?